### PR TITLE
Add changes for 1.1.1

### PR DESCRIPTION
**Pull request by xorye**
The rendered markdown for the new changes look like this:

image

Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.1.1 (October 21, 2019)
```
Comment (fbricon) [inline CHANGELOG.md]: October 23

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+
+### Enhancements
+
+ * Support glob pattern to exclude unknown validation. See [#118](https://github.com/redhat-developer/vscode-quarkus/pull/118)
+ * Preserve focus when welcome page is displayed. See [#124](https://github.com/redhat-developer/vscode-quarkus/pull/124)
+
+### Others
+
+ * Welcome page opens on the side. See [#120](https://github.com/redhat-developer/vscode-quarkus/pull/120)
```
Comment (fbricon) [inline CHANGELOG.md:8]: Should be an enhancement

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.1.1 (October 21, 2019)
+
+### Enhancements
+
+ * Support glob pattern to exclude unknown validation. See [#118](https://github.com/redhat-developer/vscode-quarkus/pull/118)
+ * Preserve focus when welcome page is displayed. See [#124](https://github.com/redhat-developer/vscode-quarkus/pull/124)
```
Comment (fbricon) [inline CHANGELOG.md]: should be a bug fix

**Review by fbricon — CHANGES_REQUESTED** [review]

**Comment by xorye** [comment]
fixed

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
...
+## 1.1.1 (October 23, 2019)
+
+### Enhancements
+
+ * Support glob pattern to exclude unknown validation. See [#118](https://github.com/redhat-developer/vscode-quarkus/pull/118)
+ * Welcome page opens on the side. See [#120](https://github.com/redhat-developer/vscode-quarkus/pull/120)
+
+### Bug Fixes
+
+ * Preserve focus when welcome page is displayed. See [#124](https://github.com/redhat-developer/vscode-quarkus/pull/124)
```
Comment (fbricon) [inline CHANGELOG.md]: should link to https://github.com/redhat-developer/vscode-quarkus/issues/73 instead

**Comment by xorye** [comment]
changed link to #73

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,16 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.1.1 (October 23, 2019)
+
+### Enhancements
+
+ * Support glob pattern to exclude unknown validation. See [#118](https://github.com/redhat-developer/vscode-quarkus/pull/118)
+ * Welcome page opens on the side. See [#120](https://github.com/redhat-developer/vscode-quarkus/pull/120)
```
Comment (fbricon) [inline CHANGELOG.md:8]: change about new command id is missing

Comment (xorye) [reply CHANGELOG.md:8]: added

**Pull request merged**